docker-compose.yml

```diff
@@ -29,7 +29,11 @@ services:
       PULSAR_PREFIX_kafkaListeners: PLAINTEXT://0.0.0.0:9092
       PULSAR_PREFIX_kafkaAdvertisedListeners: PLAINTEXT://127.0.0.1:19092
       PULSAR_PREFIX_brokerEntryMetadataInterceptors: org.apache.pulsar.common.intercept.AppendIndexMetadataInterceptor
+      PULSAR_PREFIX_kopSchemaRegistryEnable: true
+      PULSAR_PREFIX_kopSchemaRegistryPort: 8081
+      PULSAR_PREFIX_kafkaTransactionCoordinatorEnabled: true
     ports:
       - 6650:6650
       - 8080:8080
       - 19092:9092
```

Comment:
I think we can change it to `9092:9092` now. Many users might not notice the exposed port is 19092.